package.json

```diff
@@ -117,12 +117,13 @@
     "sailthru-client": "^2.0.3",
     "sharify": "^0.1.6",
     "socket.io": "^2.0.4",
-    "styled-components": "^2.1.2",
+    "styled-components": "2.4.0",
```

Comment:
Not necessary to do this now, but I think we should be able to bump this up to 3.x without problems. We did so in Force and Reaction: https://github.com/artsy/force/pull/2136/files

Reply:
Reaction wasn't bumped due to jest-styled-components issues which we wouldn't see here..

Reply:
Done! 🎉